bundle/tests/scorecard/kuttl/semeru/02-errors.yaml

```diff
@@ -0,0 +1,11 @@
+# Builds off variation 01, by disabling the semeruCloudCompiler and verifying that the Semeru deployment and service is removed
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-wsliberty-app-semeru-compiler-1
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: my-wsliberty-app-semeru-compiler-1
```

Comment:
You might also want to check that the application deployment has been changed so that it is not longer attempting to connect to the Jit Server